src/test/java/uk/gov/companieshouse/service/DissolutionPatcherTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.companieshouse.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.fixtures.DissolutionFixtures;
+import uk.gov.companieshouse.mapper.DirectorApprovalMapper;
+import uk.gov.companieshouse.mapper.DissolutionRequestMapper;
+import uk.gov.companieshouse.mapper.DissolutionResponseMapper;
+import uk.gov.companieshouse.model.db.DirectorApproval;
+import uk.gov.companieshouse.model.db.Dissolution;
+import uk.gov.companieshouse.model.dto.DissolutionCreateRequest;
+import uk.gov.companieshouse.model.dto.DissolutionCreateResponse;
+import uk.gov.companieshouse.model.dto.DissolutionPatchResponse;
+import uk.gov.companieshouse.model.enums.ApplicationStatus;
+import uk.gov.companieshouse.repository.DissolutionRepository;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class DissolutionPatcherTest {
+
+    @InjectMocks
+    private DissolutionPatcher patcher;
+
+    @Mock
+    private DissolutionRepository repository;
+
+    @Mock
+    private DissolutionResponseMapper responseMapper;
+
+    @Mock
+    private DirectorApprovalMapper approvalMapper;
+
+    @Test
+    public void patch_addsApprovalToSingleDirector_savesInDatabase_setsStatusToPendingPayment_returnsPatchResponse() {
```

Comment:
i think this can be split into multiple tests as this test is massive:
1. patch_addsApprovalToSingleDirector_savesInDatabase
2. patch_updatesStatusToPendingPayment_ifAllDirectorHaveApproved
3. patch_doesNotUpdateStatus_ifNotAllDirectorHaveApproved